### ISSUE TITLE
strange error.

### ISSUE THREAD

**Issue by BananaAcid** [vUdav/vue2-leaflet-locatecontrol#3]
My browser output. Anything works within the map.
(a featuregroup with a few markers and a polyline. as well as zooming to bounds of after 1 sec)

```
leaflet-src.js:1052 Uncaught TypeError: Cannot read property 'x' of undefined
    at Bounds.intersects (leaflet-src.js:1052)
    at i._empty (Vue2LeafletLocatecontrol.js:1)
    at NewClass._updateCircle (leaflet-src.js:12843)
    at i._updatePath (Vue2LeafletLocatecontrol.js:1)
    at i._update (Vue2LeafletLocatecontrol.js:1)
    at i._reset (Vue2LeafletLocatecontrol.js:1)
    at i.onAdd (Vue2LeafletLocatecontrol.js:1)
    at i._layerAdd (Vue2LeafletLocatecontrol.js:1)
    at NewClass.whenReady (leaflet-src.js:4458)
    at NewClass.addLayer (leaflet-src.js:6656)
```

**Comment by smesterheide** [comment]
I have the exact same error.

nuxt-leaflet: v0.0.19
vue2-leaflet: v2.2.1
leaflet: v1.5.1
vue2-leaflet-locatecontrol: v1.0.1
leaflet.locatecontrol: v0.66.2

**Comment by omrihar** [comment]
I also get the same error, using the quasar-framework. On FF I get max2 undefined, on chrome exactly the same as the ones reported above.

**Comment by smesterheide** [comment]
Fixed it with:
`import Vue2LeafletLocatecontrol from 'vue2-leaflet-locatecontrol/Vue2LeafletLocatecontrol'`

**Comment by omrihar** [comment]
Thanks! that worked for me as well :)

**Comment by mwargan** [comment]
@sambrezo did you get any weird CSS problems doing this? For me all the styling screws up...

**Comment by smesterheide** [comment]
Here is how I am using it: https://gist.github.com/sambrezo/cf0b1b78e4fbdf125aa42d4500bdec86

**Comment by vUdav** [comment]
I fix this problem in v1.0.2
